app/src/main/res/values/styles.xml

```diff
@@ -186,6 +176,16 @@
         <item name="android:textColor">?attr/primary_color</item>
     </style>
 
+    <style name="SnackbarTextView">
+        <item name="android:textColor">@color/gray200</item>
```

Comment:
^ note that we need to use a specific color here (instead of a styled color definition) because Snackbars always have a dark background, regardless of light or dark theme.